docker/dockerfiles/be/cn_entrypoint.sh

```diff
@@ -139,30 +139,8 @@ drop_my_self()
 
 exit_clean()
 {
-    log_stderr "Receives signal to exit ..."
-    pidfile=$STARROCKS_HOME/bin/cn.pid
-    if ! test -f $pidfile ; then
-        log_stderr "Can't find $pidfile!"
-    else
-        pid=`cat $pidfile`
-        if [[ "x$pid" == "x" ]] ; then
-            log_stderr "Empty pid file!"
-        else
-            log_stderr "detect cn pid $pid exists ..."
-            while true
-            do
-                if ps -p $pid &>/dev/null ; then
-                    log_stderr "cn process $pid is still alive ..."
-                    sleep $PROBE_INTERVAL
-                else
-                    log_stderr "cn process $pid dead."
-                    break;
-                fi
-            done
-        fi
-    fi
-    # remove myself from FE
-    drop_my_self $svc_name
+    log_stderr "Got SIGTERM, exit ..."
```

Comment:
this makes `exit_clean` trap useless, so a cleaner way is remove exit_clean() and the corresponding signal trap.